### ISSUE TITLE
Feedback on Charter refinement draft text

### ISSUE THREAD

**Issue by caribouW3** [w3c/process#934]
Hi,
I have been trying to apply the text proposed in section "Lifecycle of Chartered Groups" to the current chartering of a WG.
While the first part is relatively common (getting the proposal in shape, sending the advance notice, getting horizontal reviews), I've found some difficulties implementing it.

Specifically:

1.  in section "initiating...":

- Charter Facilitator is not well defined. Is it a team member? Currently our guidebook does not mandate that a single team member takes care of it until the end. It is very common that more than 1 team member working on a (re)chartering. Or is it an external person (or several) proposing the new charter (group chair? CG chair? another person? Again there is often several people involved...

- Team rejection is not defined. In general, I think that the team will delay the start of the chartering process (i.e. the advance notice) until there's a reasonably well-constructed proposal. The guidebook does not mention anything that could be referred to as a rejection.

2. In section "Charter Refinement"

- Charter facilitator is still unclear, although he's given the confusing role of a Chair person. What does chairing mean in a process where no group is formed and no meeting happens? I find the term Chair in that context very confusing and not useful. Why not sticking with the term "facilitator" and introduce a new one that's confusing ?

- issues must be "formally addressed". By whom? Who's responsible for evaluating the consensus of the resolutions?

- "the group is made up of all individuals participating in this process" : the "group" here is undefined. what does "participating" means in that context? Anyone who sends a comment? anyone proposing a resolution?  the term "group" usually refers to something implemented with a process for joining/leaving. Is it the same intent here? AC and Team (and public?) individuals joining a formal "group" right after the advance notice?

- "chair decision to abandon" seems to imply that the chair has super power to drop the case. That would be rather strange, in particular for a WG rechartering, to give such power to a single person. 

- the next NOTE is very hard to parse (disclaimer: I'm not a native speaker) and understand. Also, "misbehavior" is undefined. e.g. Failure to evaluate consensus is not misbehavior... I think.

Finally, I had expected to see the Disposition of Comments sent to the AC prior to the vote, not at the same time, in order to get more opinions on the substantial issues before the vote instead of getting them during the vote. It seemed to me that the problem to solve with all this new wording was to add a milestone to get AC reps involved earlier. If not, why is all that wording in the process document instead of the Guide?

**Comment by frivoal** [comment]
Thanks for the feedback. Here's an attempt to answer your questions.

As a general note, there's nothing yet (as far as I know, though I could be wrong) in the Guide about this, since this is a new proposed procedure, not yet an official one, so it is unsurprising that you wouldn't at the moment find detailed guidance for how to roll this out. I'd suggesting syncing up with @plehegar on that.

> Currently our guidebook does not mandate that a single team member takes care of it until the end. It is very common that more than 1 team member working on a (re)chartering. Or is it an external person (or several) proposing the new charter (group chair? CG chair? another person? Again there is often several people involved...

That is a deliberate change. There can absolutely be several people (from the Team) working on a charter, but there should be an identified person whose responsibility is being in charge of consensual decisions. We want to separate the role of the person or persons advocating (and editing) a charter draft from the one who checks if there's consensus and calls issues resolved. It's not forbidden that this would be the same person, just like a WG chair can also be the editor of the spec, but it's not generally expected to be the same person. (see also later questions about the facilitator and their role).

The (draft) Process is also deliberately silent on who this person is, so that the Team can make a choice.

> Team rejection is not defined.

If the Team is considering proposing a charter on its own initiative, it can wait for a long as it wants before initiative charter development, or change its mind and not start after all, with no recorded decision and no formalism as long as the charter review notice has not yet been sent. Same thing if the Team was considering a charter based on an *informal* suggestion by somebody.
However:
* if there was a formal request from a Member that the Team initiate charter refinement, then the Team must answer formally (and if the answer is no, the requester might Formally Object to that). Reasons for rejecting are up to the Team, but "not incubated enough, wouldn't succeed, so please prepare more" or "fuel composition for combustion engines is way out of scope for W3C, you need to propose this elsewhere" would be reasonable and could be expected to be supported by a Council in case of Formal Objection, while "I don't like you" or "I don't feel like it this year" would not be reasonable, and would be expected to be overruled by a Council.
* if the charter review notice has been sent, then it's up to the group facilitator to decide whether we should give up, which they shouldn't do on a while, otherwise there will likely be formal objections against this arbitrary decision to close. However, if there is no consensus and no reasonable path forward, there may be times where abandoning is the right call.

> Charter facilitator is still unclear, although he's given the confusing role of a Chair person. What does chairing mean in a process where no group is formed and no meeting happens? I find the term Chair in that context very confusing and not useful. Why not sticking with the term "facilitator" and introduce a new one that's confusing ?

The goal is not to say that the facilitator also has the role the chair, but to say that facilitator _is_ a kind of chair. We're calling them facilitator in an attempt to avoid confusion with the person who will eventually be chair of the WG. But we're defining them as being a type of chair so that they have the same powers as normally vested into a chair. See also the next question.

> Issues must be "formally addressed". By whom? Who's responsible for evaluating the consensus of the resolutions?

It is the facilitator's role is to evaluate consensus and to make sure issues are addressed. That's why we have this role in the first place.

> "the group is made up of all individuals participating in this process" : the "group" here is undefined. what does "participating" means in that context? Anyone who sends a comment? anyone proposing a resolution?

Yes, anyone who raises an issues, comments on one, proposes a solution… is participating. It doesn't mean a chair needs a response from everybody (just like a WG chair doesn't need a response from every member of the WG). But if someone has a problem with something, it's not appropriate to just close the issue with "sorry, that's not what we're after", without consulting anyone. For each topic, the resolution should reflect the consensus of those who are participating in the discussion (and if no consensus can be found, the usual rules for [managing dissent](https://www.w3.org/policies/process/drafts/#managing-dissent) or [voting](https://www.w3.org/policies/process/drafts/#Votes) apply.

> "chair decision to abandon" seems to imply that the chair has super power to drop the case. That would be rather strange, in particular for a WG rechartering, to give such power to a single person.

Until now, the Team collectively has had that power. If the Team chose to give up on a charter, then the charter dies, and there's nothing anybody can do about it. Framing it this way makes it into a identifiable decision, which means it can be objected to. I don't expect the facilitator to randomly drop charters (including recharters) that should progress, but if they do drop one, it is now something members can push back against.

> the next NOTE is very hard to parse (disclaimer: I'm not a native speaker) and understand. Also, "misbehavior" is undefined. e.g. Failure to evaluate consensus is not misbehavior... I think.

You are referring to this text and the accompanying note:

> Any [Formal Objection](https://www.w3.org/policies/process/drafts/#FormalObjection) filed during the [charter refinement](https://www.w3.org/policies/process/drafts/#charter-refinement) phase—​other than an objection to the choice of [Chartering Facilitator](https://www.w3.org/policies/process/drafts/#chartering-facilitator) or to a decision to abandon the proposal—​is not considered [registered](https://www.w3.org/policies/process/drafts/#registering-objections) until the close of the [Advisory Committee Review](https://www.w3.org/policies/process/drafts/#advisory-committee-review) of the charter, and is registered against that [W3C Decision](https://www.w3.org/policies/process/drafts/#def-w3c-decision).
>
> Note: This enables all [Formal Objections](https://www.w3.org/policies/process/drafts/#FormalObjection) on the same proposed [charter](https://www.w3.org/policies/process/drafts/#charter) to be handled together, while allowing [Formal Objections](https://www.w3.org/policies/process/drafts/#FormalObjection) to [Chartering Facilitator](https://www.w3.org/policies/process/drafts/#chartering-facilitator) misbehavior that the [Team](https://www.w3.org/policies/process/drafts/#team) is failing to address to be processed immediately.

The Note may need rephrasing. What this is trying to do/say is:
* if there are formal objections raised about the content of the charter, about how well points of views were taken into account when determining consensus, and so on, all these formal objections are handled together at the end, so that the Council can look at the whole picture, and also, before we get to that point, to give a chance to further discussions to resolve the objections.
* however, in the following two cases, the formal objections are treated without waiting:
    * the facilitator makes a decision to give up: we can't wait until the AC review, since there won't be one.
    * the choice of facilitator is itself called into question, by participants who don't believe they're capable of, or trying to, do a balanced job of evaluating consensus and addressing issues. Formally objecting to that is very heavy handed, and if there's a problem, informal discussions with the Team are more likely to be productive. But it is generally possible to object to a choice of chair (here or for regular WG/IG) anyway, so if it does happen, it is better to process it sooner rather than later.

> Finally, I had expected to see the Disposition of Comments sent to the AC prior to the vote, not at the same time, in order to get more opinions on the substantial issues before the vote instead of getting them during the vote. It seemed to me that the problem to solve with all this new wording was to add a milestone to get AC reps involved earlier. If not, why is all that wording in the process document instead of the Guide?

The charter review notice is what is sent to the AC prior to the vote in order to get opinions and involvement. It is also allowed to bring it up the the AC's attention multiple times, if the facilitator thinks that they're not getting quite enough feedback (this can be seen as part of the requirement to get wide review). A DoC is not a document that solicits feedback, but one that documents how much solicitation has occurred already, and how the feedback was handled. Informing AC Reps for the AC-Review on the amount of feedback received and about the extend to which issues which were raised have found consensual conclusions is useful to let them know how much scrutiny is appropriate.

**Comment by caribouW3** [comment]
> As a general note, there's nothing yet (as far as I know, though I could be wrong) in the Guide about this, since this is a new proposed procedure, not yet an official one, so it is unsurprising that you wouldn't at the moment find detailed guidance for how to roll this out. I'd suggesting syncing up with @plehegar on that.

This issue comes from non-conclusive discussion that I've had with him, implementation details can't be clearly stated if the overall intent of those sections is not clear. What is exactly the problem this is trying to solve? Participation level? amount of late FO? It seems that those could be improved with more efficient communication. FWIW almost all councils so far would not have been avoided by having such a process.

I understand that there is a desire to rely on the existing chair/group definitions, but the analogy makes the wording very confusing. Thank you for clarifying. I still see several gaps with the Facilitator role definition:

1. IIUC the Facilitator can't be the Team member who helps the proposal to get in shape and edits the charter, the team contact, nor it can be a participant (e.g. WG or CG participants). Such a powerful Facilitator has to be obviously neutral [Currently, consensus checking is done by a (reasonably large and diverse) set of Team members.]
 
> > "chair decision to abandon" seems to imply that the chair has super power to drop the case. That would be rather strange, in particular for a WG rechartering, to give such power to a single person.
> 
> Until now, the Team collectively has had that power. If the Team chose to give up on a charter, then the charter dies, and there's nothing anybody can do about it. Framing it this way makes it into a identifiable decision, which means it can be objected to. I don't expect the facilitator to randomly drop charters (including recharters) that should progress, but if they do drop one, it is now something members can push back against.

Side note: Members can already propose to recharter as a "push back" in an issue raised by the Team to close a WG after the end of its current charter. (It happened last year for the SVG WG). Therefore there seems to be no need for additional process for this case.

2. Formal Objections can happen only after formally recorded resolutions, which happen usually after a vote (or CfC). There are generally no votes/CfC during the charter editing/review period before the AC review, so in fact this new refinement phase introduces a process similar to AC review (or group decision), likely with only a subset of the AC+non-AC participants, with a Disposition of Comments as a result. Is that right?
 In most cases there would still be no vote needed, However, it seems that again the Facilitator, being the one to decide whether to hold a vote, has a lot of decision power. If the Facilitator decides not to have a vote, the FO would then be a request  to change the Facilitator. 

Then... who's going to judge whether we need to change the Facilitator? Who's deciding who the Facilitator is, BTW?

It seems to me that the Facilitator might just be another level of indirection and unnecessary complexity added to the chartering process.

**Comment by ianbjacobs** [comment]
Formal Objections are a big hammer and imply significant resource implications. Rather than introduce new hooks for Formal Objections into the process, there may be other ways to achieve goals, in particular:

* Find a mechanism to ensure all charters receive AC review, including those not supported by the staff.
* Increase transparency about staff decisions. 

For example (and without needing to change the W3C Process):

* For any charter sent to the AC for review, the staff (e.g., coordinated by the head of strategy) could provide commentary about the charter.
* Where the staff does not support a charter, the commentary would provide rationale. 

That should suffice to achieve both goals. 

Overall I think we should endeavor to shrink and simplify the W3C Process. We should evolve and document best practices outside of the Process Document. Many of the other suggestions in the proposed additions to the Process Document do not need to be formally specified. They can be promoted (with good tooling, templates, etc.) as best practices, until they become the obvious way to do things. Let's solidify our culture and simplify our process.

**Comment by fantasai** [comment]
@ianbjacobs The goal of this is actually to decrease the formal objections, by having an earlier discussion phase that is clearly defined -- and if a charter goes to AC Review with objections, by batching those objections with the results of the AC review at the end of that process. We can't do this batching unless the Process defines it.

I think your suggestions are all good improvements to the (lower-case) chartering process, though.

**Comment by plehegar** [comment]
Thank you @caribouW3 for the feedback. We think we addressed your issues as best as we can, with @ianbjacobs help. Closing this issue for now. feel free to open new issues if you believe some parts remain problematic.
